README.md

```diff
@@ -1 +1,100 @@
 # Custom Enumerable Project
+
+This repository complements the
+[Custom Enumerables Project](https://www.theodinproject.com/paths/full-stack-ruby-on-rails/courses/ruby-programming/lessons/custom-enumerables)
+in the Ruby Course for The Odin Project.
```

Comment:
Yes, sorry! I was trying to use semantic line breaks (https://sembr.org/) since it also makes it easier to work with in my text editor and it'll render the same, but I understand that it could be hard to read and I'll revert it.